src/databricks/labs/ucx/source_code/jobs.py

```diff
@@ -138,10 +138,10 @@ def _register_library(self, graph: DependencyGraph, library: compute.Library) ->
                 for requirement in contents.splitlines():
                     clean_requirement = requirement.replace(" ", "")  # requirements.txt may contain spaces
                     if clean_requirement.startswith("-r"):
-                        logger.warning(f"References to other requirements file is not supported: {requirement}")
+                        logger.warning(f"References to other requirements files are not supported: {requirement}")
```

Comment:
It is singular, right?

Reply:
The subject is _References_, plural. Either of the following is correct:

 - _Reference_ to other requirements file is not supported: {requirement}
 - References to other requirements files _are_ not supported: {requirement}

The meaning is subtly different (general vs specific) but either would be appropriate.

Reply:
> Reference to other requirements file is not supported: {requirement}

Keep that one!